### PR TITLE
Don't use the Ingress cert when running on IBM Cloud

### LINKED ISSUE #2454: Don't use Ingress certificate for external MCG communications when running on IBM Cloud
The default root certificates are sufficient. The Ingress cert leads to a verification failure.

### PR DESCRIPTION
**Pull request by MeridianExplorer**
Using the Ingress certificate causes an SSL verification failure.
The root certificates are enough, and thus, we rely on them instead.

Also improved the `_retrieve_data_reduction` flow.

Closes #2454.

### CODE REVIEW

**Comment by ebenahar** [comment]
@MeridianExplorer Thanks for fixing! shouldn't we maintain an issue for the problem you solved here?

**Comment by MeridianExplorer** [comment]
@ebenahar Done

**Comment by MeridianExplorer** [comment]
[Verified](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/10832/)

File: ocs_ci/ocs/resources/mcg.py [hunk 1]
```diff
@@ -349,7 +329,8 @@ def _retrieve_reduction_data():
                     )
         except TimeoutExpiredError:
             logger.error(
-                'Not enough data reduction. Something is wrong.'
+                'Data reduction is insufficient. '
+                f'{total_size - total_reduced} bytes reduced.'
```
Comment (bernerhat) [inline ocs_ci/ocs/resources/mcg.py:333]: i would have added the amount expected as well

**Pull request merged**